test/index.js

```diff
@@ -35,29 +37,19 @@ perform: true
           }}))
       },
       issues: {
-        getComments: expect.createSpy().andReturn(Promise.resolve(
```

Comment:
so metadata takes care of this, but we don't need to mock it?

Reply:
Ah, so pulling the issue pulls the OP comment, which gets you the metadata. That's.... Awesome.